docs/aws/karpenter.cloudformation.yaml

```diff
@@ -0,0 +1,88 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Resources used by https://github.com/awslabs/karpenter
+Parameters:
+  OpenIDConnectIdentityProvider:
+    Type: String
+    Description: "Example oidc.eks.us-west-2.amazonaws.com/id/1234567890. "
+Mappings:
+  ServicePrincipals:
+    aws-cn:
+      ec2: ec2.amazonaws.com.cn
+    aws-us-gov:
+      ec2: ec2.amazonaws.com
+    aws:
+      ec2: ec2.amazonaws.com
```

Comment:
I'm pretty sure you can remove this and instead use something like `!Sub "ec2.${AWS:URLSuffix}"` (see [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html)) inline where you need to.

Reply:
🙏